src/__tests__/components/molecules/Header.test.tsx

```diff
@@ -1,12 +1,34 @@
-import { render, screen } from '@testing-library/react';
+import { render, waitFor } from '@testing-library/react';
 import { NHSNotifyHeader } from '@molecules/Header/Header';
+import { Authenticator } from '@aws-amplify/ui-react';
+import { usePathname } from 'next/navigation';
+
+jest.mock('next/navigation');
 
 describe('Header component', () => {
-  it('renders component correctly', () => {
-    render(<NHSNotifyHeader />);
+  it('auth path', async () => {
+    jest.mocked(usePathname).mockReturnValue('/auth');
+
+    const container = render(
+      <Authenticator.Provider>
+        <NHSNotifyHeader />
+      </Authenticator.Provider>
+    );
+
+    await waitFor(() => {});
```

Comment:
why the empty waitfor?

Reply:
it stops jest giving you warnings about wrapping everything in an `act()` block